### ISSUE TITLE
Resolution

### ISSUE THREAD

**Issue by twobleak** [NCAR/pi-wrf#5]
Hi

I think the default resolution for a model run is 30km? Is it possible to change this at all? I could see some reference to the resolution in the config files within the image, but was unsure if this could be modified at all...

**Comment by wefoust** [comment]
You are correct, the default run is 30km. At the moment, it is not trivial to change the resolution. However, this is a highly requested feature and this will be incorporated into the next version of the app. 

Even in the next version, there still may be considerable limits due to hardware constraints, need for high res topography, and other configuration changes. Is there a particular resolution and or domain you are interested in?

**Comment by twobleak** [comment]
It would be interesting to see just how much performance could be squeezed out of the Pi for this, a 30km (48hour) run with ~1000 grid points actually runs pretty quickly on the Pi4 4gb model. It was this speed that got me thinking that there might be room for higher resolutions/different physics options etc - for those that don't mind waiting a bit longer :)

Have you tested any higher resolutions at all?  I wasn't expecting the Pi to be capable of very high resolutions/high res topography as I appreciate the hardware is inevitably limited but perhaps some different options could be useful?
